__tests__/app.test.js

```diff
@@ -238,4 +261,54 @@ describe("GET", () => {
         });
       });
   });
+
+  test("/api/articles/:article_id/comments 200: returns the comments of the specific article as requested by the client.", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual({
+          commentsOfThisArticle: [
+            {
+              comment_id: 11,
+              body: "Ambidextrous marsupial",
+              article_id: 3,
+              author: "icellusedkars",
+              votes: 0,
+              created_at: "2020-09-19T23:10:00.000Z",
+            },
+            {
+              comment_id: 10,
+              body: "git push origin master",
+              article_id: 3,
+              author: "icellusedkars",
+              votes: 0,
+              created_at: "2020-06-20T07:24:00.000Z",
+            },
+          ],
+        });
+      });
+  });
+
+  test("/api/articles/:article_id/comments 410: returns an error if the article doesn't exist.", () => {
+    return request(app)
+      .get("/api/articles/123/comments")
+      .expect(410)
+      .then(({ body }) => {
+        expect(body).toEqual({
+          message: "??? There isn't an article stored in this parameter.",
+        });
+      });
+  });
+
+  test("/api/articles/:article_id/comments 403: returns an error if there are no comments in the article.", () => {
+    return request(app)
+      .get("/api/articles/12/comments")
+      .expect(403)
+      .then(({ body }) => {
+        expect(body).toEqual({
+          message: "There are no comments yet on this article.",
+        });
+      });
+  });
```

Comment:
There is an error handling test missing. What if the article id is not a number?

Reply:
the test already exists, it expects the error to equal 
`{
	"message": "Unknown endpoints are not acceptable on this platform."
}`